sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_automation_client_async.py

```diff
@@ -127,6 +129,7 @@ def from_connection_string(
 
         return cls(endpoint, access_key, **kwargs)
 
+    @distributed_trace
```

Comment:
one more thing - is there reason to add this here?
from doc says : https://azure.github.io/azure-sdk/python_implementation.html#distributed-tracing

"DO create a new span for each outgoing network call. If using the HTTP pipeline, the new span is created for you."

as this is not network call (its just getting client), would this need distributed trace?

Reply:
yea not sure. Pylint complained about it. Should we ignore it?

Reply:
checked with other ACS team - they seem to include distributed tracing for even non-network call methods. we can probably keep as is then.